packages/moonstone/GridListImageItem/GridListImageItem.js

```diff
@@ -51,6 +51,15 @@ const GridListImageItemBase = kind({
 	name: 'GridListImageItem',
 
 	propTypes: /** @lends moonstone/GridListImageItem.GridListImageItemBase.prototype */ {
+		/**
+		 * The voice control intent.
+		 *
+		 * @type {String}
+		 * @default 'Select'
+		 * @public
+		 */
+		'data-webos-voice-intent': PropTypes.string,
```

Comment:
Normally we would need `@memberof` here but I just upgraded our doc parsing tool so this is no longer required!  :partyparrot: